packages/minifiers/minifiers.js

```diff
@@ -109,7 +109,7 @@ var rewriteRules = function (rules) {
     // a folder starting with "/packages/" is served on the same path as
     // it was on the filesystem and everything else is served on root "/".
     if (! basePath.match(/^\/?packages\//i))
-        basePath = "/";
+        basePath = (process.env.ROOT_URL_PATH_PREFIX || "") + "/";
```

Comment:
@glasser, process.env contains **all** environment variables. So as soon as you set it before running meteor, it'll be there.

Proof:
```
$ ABC="testofenvvar" meteor
[[[[[ ~/meteor/app ]]]]]

=> Started proxy.
=> Started MongoDB.
=> Started your app.

=> App running at: http://localhost:3000/
```

Then:
```
$ meteor shell
<snip>
> process.env.ABC
'testofenvvar'
>
```